azuredevops/internal/acceptancetests/resource_servicehook_webhook_tfs_test.go

```diff
@@ -0,0 +1,215 @@
+//go:build (all || resource_servicehook_webhook_tfs) && !exclude_servicehook
+
+package acceptancetests
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/servicehooks"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+)
+
+func TestAccServicehookWebhookTfs_basic(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	url := "https://example.com/webhook"
+
+	resourceType := "azuredevops_servicehook_webhook_tfs"
+	tfCheckNode := resourceType + ".test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: CheckServicehookWebhookTfsDestroyed,
+		Steps: []resource.TestStep{
+			{
+				Config: testutils.HclServicehookWebhookTfsResourceWithGitPushEvent(projectName, url),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(tfCheckNode, "project_id"),
+					resource.TestCheckResourceAttr(tfCheckNode, "url", url),
+					resource.TestCheckResourceAttr(tfCheckNode, "git_push.#", "1"),
```

Comment:
These won't be needed if we have a import step after each test step.

Reference: https://github.com/microsoft/terraform-provider-azuredevops/blob/2dab4c90d03189c451e1a47d192b433e18973b96/azuredevops/internal/acceptancetests/resource_project_test.go#L39-L44